src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -82,12 +82,15 @@
   }
 
   /**
-   * Create a client builder with a basic string transcoder.
-   * @param charset the charser to use for encoding/decoding strings
+   * Create a client builder with a basic string transcoder using the UTF-8
+   * Charset. To build a client with a different charset, provide a StringTranscoder
+   * constructed with a different charset in the public MemcacheClientBuilder
```

Comment:
Having to explain this indicates a problem with the API. How about adding an overloaded method without the charset (I agree on promoting UTF-8 as the "default" charset) while keeping the existing one?